packages/atomic/src/components/common/suggestions/suggestions-events.ts

```diff
@@ -0,0 +1,76 @@
+import type {HTMLStencilElement} from '@stencil/core/internal';
+import type {LitElement} from 'lit';
+import {closest} from '../../../utils/dom-utils';
+import {buildCustomEvent} from '../../../utils/event-utils';
+import type {AnyBindings} from '../interface/bindings';
+import type {SearchBoxSuggestionsEvent} from './suggestions-types';
+
+/**
+ * Supported search box element selectors.
+ */
+const searchBoxElements = [
```

Comment:
I personally prefer to keep exported stuff first in a module, and keep the private stuff at the end.

Also I'd remove the comments that were added above these private declarations. IMO they're self-documenting and the comments add no value.